libc/src/sys/mman/linux/remap_file_pages.cpp

```diff
@@ -0,0 +1,44 @@
+//===----- Linux implementation of the POSIX remap_file_pages function ----===//
```

Comment:
nit: since this function isn't actually from POSIX, you can remove that from the title here.